server/zip.js

```diff
@@ -0,0 +1,83 @@
+const ApiKeys = require('../config/api-config');
+const client = require('twilio')(ApiKeys.twilioAccountSid, ApiKeys.twilioAuthToken);
+const twilio = require('twilio');
+const Promise = require('bluebird');
+const request = Promise.promisifyAll(require('request'), { multiArgs: true });
+
+const askForZipCode = (number, name) => {
+  client.messages.create({
+    to: number,
+    from: ApiKeys.twilioNumber,
+    body: 'Hey Gary, ' + 'text me your zip code and I can forecast the weather. Just like this: 94102. Give it a try :)'
+  }, (err, message) => {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log(message.sid);
+    }
+  })
+}
+
+const getEachNum = (contacts, callback) => {
+  for (let name of Object.keys(contacts)) {
+    callback(contacts[name], name);
+  }
+}
+// getEachNum(ApiKeys.testContacts, sendSillyQnAToNums);
+getEachNum(ApiKeys.testContacts, askForZipCode);
+
+const convertToFahrenheit = function (degrees) {
+  return (degrees - 273) * 9 / 5 + 32;
+}
+
+const getWeatherForecast = (zipCode) => {
+    return new Promise( (resolve, reject) => {
+      request.get('http://api.openweathermap.org/data/2.5/forecast?zip=' + zipCode + ',us&appid=' + ApiKeys.openWeatherApiKey, (err, res, body) => {
```

Comment:
Surprised that you're not using template literals when concating your string here considering your ES6 syntax throughout. May want to consider adding it.